micropub.php

```diff
@@ -369,7 +390,7 @@ private static function update() {
 				static::error( 400, 'replace must be an object' );
 			}
 			foreach ( static::mp_to_wp( array( 'properties' => $replace ) )
-					  as $name => $val ) {
+				as $name => $val ) {
```

Comment:
this seems wrong. it should be in two spaces to show that it's inside the paren above, right?